chore: Plural data sources in TPF leveraging on resource data source (#2844)
mongodb/terraform-provider-mongodbatlas: docs/data-sources/encryption_at_rest_private_endpoints.md

### Required

- `cloud_provider` (String) Label that identifies the cloud provider for the Encryption At Rest private endpoint.
- `project_id` (String) Unique 24-hexadecimal digit string that identifies your project.

### Read-Only

> - `results` (Attributes List) List of returned documents that MongoDB Cloud provides when completing this request. (see [below for nested schema](#nestedatt--results))

Reviewer: ```suggestion
- `results` (Attributes List) List of documents that MongoDB Cloud returns for this request. To learn more, see the following [nested schema](#nestedatt--results).
```
